Wrappers/Python/cil/plugins/astra/operators/AstraProjector3D.py

```diff
@@ -1,30 +1,25 @@
-#========================================================================
-# Copyright 2019 Science Technology Facilities Council
-# Copyright 2019 University of Manchester
+# -*- coding: utf-8 -*-
+#  Copyright 2019 - 2022 United Kingdom Research and Innovation
```

Comment:
Most of the headers are with `2018--2022`. Here it is from `2019`, when we wrote this script wiht these operators. Maybe we can change it to `2018`.

Reply:
I think it has to reflect the date when the file has been created. The copyright cannot cover before the date of creation of the actual file, can it?